### ISSUE TITLE
Login not successful

### ISSUE THREAD

**Issue by haoshu1995** [MiniGlome/Archive.org-Downloader#80]
The auto-login of this script is not getting respond from the website, is it because of problem in my Internet?

**Comment by MiniGlome** [comment]
Probably, what command did you use?

**Comment by OmarAmirAhamada** [comment]
I hope ou still provide support for this, i wrote the command  and it output this :
" 1 Book(s) to download
[-] Invalid credentials!
"
i checked the credentials thoroughly i am certain they are correct 
this is my command line i will copy it and paste it here

python3 archive-org-downloader.py -e machadoramo156@gmail.com -p AmG8?PfS"fhNkX" -r 0 -u https://archive.org/details/workbookforcriti0000epst

**Comment by MiniGlome** [comment]
You cannot use quotes in your password or you will have to escape them in the command line.

**Comment by J-Ligas** [comment]
> You cannot use quotes in your password or you will have to escape them in the command line.

yep, you are right, i changed the password for a simple one without special characters and it worked, but once it got downloaded the content was filled with "Page temporarily unavailable" :,(
